ample/util/process_models.py

```diff
@@ -142,7 +143,9 @@ def check_models_dir(models_in_dir, models_out_dir):
     """Examine a directory of PDB files to determine their suitability for running with AMPLE."""
     assert os.path.isdir(models_in_dir)
     pdb_structures = glob.glob(os.path.join(models_in_dir, "*.pdb"))
-    pdb_structures += glob.glob(os.path.join(models_in_dir, "*.PDB"))
+    # glob.glob in Windows is not case sensitive
+    if not sys.platform.startswith("win"):
+        pdb_structures += glob.glob(os.path.join(models_in_dir, "*.PDB"))
```

Comment:
Do I understand that what you want is to catch all files with the pdb extension regardless of lower case or upper case both in unix and in windows? Then why not using:
`glob.glob(os.path.join(models_in_dir, '*[pP][dD][bB]'))` 
For sure this works on linux, I can't test it on windows but I can't see why it wouldn't work.

Reply:
That's a fair shout - I thought it could be done more simply but was being lazy. I'll test it in a bit